### PR TITLE
Dx 1883 added integration tests

### PR DESCRIPTION
**Pull request by jmulford-bw**

### CODE REVIEW

**Review by ajrice6713 — APPROVED** [review]

File: tests/messaging.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            from: process.env.PHONE_NUMBER_OUTBOUND,
+            text: 'TS messaging test'
+        };
+
+        const createMessageResponse = await controller.createMessage(accountId, body);
+        expect(createMessageResponse.result.applicationId).toBeDefined();
+    });
+    
+
+    it('should throw error on invalid values', async () => {
```
Comment (hamermike) [inline tests/messaging.test.js:30]: I'd recommend a single incorrect value in the request to make the test more explicit. Maybe this becomes an "invalid application id" test.

File: tests/messaging.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        const accountId = process.env.BANDWIDTH_ACCOUNT_ID;
+        const body = {
+            applicationId: process.env.MESSAGING_APPLICATION_ID,
+            to: [process.env.PHONE_NUMBER_INBOUND],
+            from: process.env.PHONE_NUMBER_OUTBOUND,
+            text: 'TS messaging test'
+        };
+
+        const createMessageResponse = await controller.createMessage(accountId, body);
+        expect(createMessageResponse.result.applicationId).toBeDefined();
```
Comment (hamermike) [inline tests/messaging.test.js:26]: I'd do a comparison to validate that the returned application id is the expected one and toss a few more expectations in there for the rest of the response values.

**Review by hamermike — CHANGES_REQUESTED** [review]

File: tests/messaging.test.js (hunk 1 above)
Comment (jmulford-bw) [reply tests/messaging.test.js]: Good catch, actually the other SDK tests have an invalid "to" number so I'll just change it to that for consistency. Updated

File: tests/messaging.test.js (hunk 2 above)
Comment (jmulford-bw) [reply tests/messaging.test.js:26]: Oh you're right, we can check the application ID since that's something that will be consistent. Updated

Comment (hamermike) [reply tests/messaging.test.js]: I'd add the rest of the expected response fields in there as well. Might as well check everything in one swoop!

Comment (jmulford-bw) [reply tests/messaging.test.js]: Good callout, done 🎉

**Review by hamermike — APPROVED** [review]

**Pull request merged**